source/managing-teams/holding-one-on-ones.html.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Holding 1:1s
+meta_description: >
+  One of the responsibilities of an Engineering Manager is to hold regular 1:1s with their team,
+  to exchange feedback and ensure they're happy.
+---
+
+All Engineering Managers are expected to have recurring 1:1 meetings (one-on-ones) with their team.
+These meetings are typically bi-weekly and 30-minutes long (although the frequency and length are
+decided with the teammate), and both parties can contribute to the agenda, which may or may not have
+fixed points and sections.
+
+One-on-ones are possibly the most valuable tool at a manager's disposal. They create a fixed cadence
+through which you can carry out a lot of your responsibilities, such as:
+
+- Gauging the teammate's morale and overall satisfaction with Nebulab.
+- Supporting the teammate in their personal and professional growth.
+- Providing and receiving positive and constructive feedback.
+- Keeping your teammate in the loop about what's going on in the company.
+- Having performance and compensation-related conversations.
+
+No matter your workload and priorities, 1:1s should be considered a sacred ritual. It's okay to
+reschedule a 1:1 every once in a while if absolutely needed—it's NOT okay to skip them.
```

Comment:
Space before and after `–`?

Reply:
Nope, the recommendation is actually not to use any spaces.

Reply:
gotcha